assets/prometheus-k8s/prometheus.yaml

```diff
@@ -89,6 +90,18 @@ spec:
     - --label=namespace
     image: quay.io/coreos/prom-label-proxy:v0.1.0
     name: prom-label-proxy
+  - args:
```

Comment:
I don't think this is necessary. Prometheus reloads the [scape managers every 5 seconds](https://github.com/prometheus/prometheus/blob/b41d6d54f2262f089ff1b7151f67a00fcdfb8e0f/scrape/manager.go#L96), which reloads all scrape pools, and reloading a scrape pool [re-creates the http client](https://github.com/prometheus/prometheus/blob/b41d6d54f2262f089ff1b7151f67a00fcdfb8e0f/scrape/scrape.go#L267), which also re-[reads the TLS configuration](https://github.com/prometheus/prometheus/blob/b41d6d54f2262f089ff1b7151f67a00fcdfb8e0f/vendor/github.com/prometheus/common/config/http_config.go#L255:6). 5 seconds delay is acceptable I think as kubelets reloading their certificates and the secret being re-mounted is racy anyways.

Reply:
TIL!

Reply:
fwiw, let's persist this fact in a comment inside jsonnet, otherwise it is not obvious why a configmap reloader is not needed.

Reply:
done and done